### PR TITLE
ci: ignore draft pr

### LINKED ISSUE #502: build: add a workflow that suspends CI if PR is draft
Draft a PR means something is not ready, to reduce the CI workload, it's better to suspend the CI until it's ready.

### PR DESCRIPTION
**Pull request by FeelyChau**
Fix #502

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,6 +21,7 @@ on:
 jobs:
   universal:
     name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    if: ${{ github.event.pull_request.draft == false }}
```
Comment (yorkie) [inline .github/workflows/build.yml:24]: This will make it broken with push event

**Review by yorkie — CHANGES_REQUESTED** [review]

**Comment by yorkie** [comment]
And if we were to fix an issue at GitHub, it's recommended to name your branch: "fix/gh-{issue_id}".

**Comment by FeelyChau** [comment]
> gh

What dose the 'gh' mean for?

File: .github/workflows/build.yml (hunk 1 above)
Comment (FeelyChau) [reply .github/workflows/build.yml]: Ruled out the event pull.

**Comment by yorkie** [comment]
> > gh
> 
> What dose the 'gh' mean for?

gh = github

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ on:
     - 'notebooks/**'
     - '.github/markdown.yml'
   pull_request:
+    types: [ pened, synchronize, reopened, review_requested, ready_for_review ]
```
Comment (yorkie) [inline .github/workflows/build.yml:12]: "pened" -> "opened"?

Comment (yorkie) [reply .github/workflows/build.yml:12]: "review_requested" should not be triggered for build.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -20,6 +21,7 @@ on:
 jobs:
   universal:
     name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    if: ${{ github.event_name == 'pull' || github.event.pull_request.draft == false }}
```
Comment (yorkie) [inline .github/workflows/build.yml:24]:
```suggestion
    if: ${{ github.event_name == 'push' || github.event.pull_request.draft == false }}
```

File: .github/workflows/markdown.yml [hunk 4]
```diff
@@ -4,11 +4,13 @@ on:
     branches:
     - master
   pull_request:
+    types: [ pened, synchronize, reopened, review_requested, ready_for_review ]
     branches:
     - master
 
 jobs:
   markdown-link-check:
+    if: ${{ github.event_name == 'pull' || github.event.pull_request.draft == false }}
```
Comment (yorkie) [inline .github/workflows/markdown.yml:13]:
```suggestion
    if: ${{ github.event_name == 'push' || github.event.pull_request.draft == false }}
```

File: .github/workflows/markdown.yml [hunk 5]
```diff
@@ -4,11 +4,13 @@ on:
     branches:
     - master
   pull_request:
+    types: [ pened, synchronize, reopened, review_requested, ready_for_review ]
```
Comment (yorkie) [inline .github/workflows/markdown.yml:7]: Ditto.

**Review by yorkie — CHANGES_REQUESTED** [review]

**Pull request merged**